pkg/cli/list.go

```diff
@@ -25,6 +25,7 @@ var (
 	focusDir          focusDirection
 	exposureAnalysis  bool
 	explain           bool
+	explainOn         explainOnly
```

Comment:
rename `explainOn` to `explainOnlyAllowOrDeny`

Reply:
done